bigbone/src/main/kotlin/social/bigbone/api/entity/admin/AdminDimension.kt

```diff
@@ -0,0 +1,125 @@
+package social.bigbone.api.entity.admin
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents qualitative data about the server.
+ * @see <a href="https://docs.joinmastodon.org/entities/Admin_Dimension/>Mastodon API Admin::Dimension documentation</a>
```

Comment:
This line needs a `"` at the end of the link.

Reply:
Done 👍